packages/components/src/tabs/styles.module.scss

```diff
@@ -71,37 +66,41 @@ export const StyledTabList = styled( Ariakit.TabList )`
 			width: calc( var( --antialiasing-factor ) * 1px );
 			transform: translateX( calc( var( --selected-start ) * var( --direction-factor ) * 1px ) )
 				scaleX( calc( var( --selected-width, 0 ) / var( --antialiasing-factor ) ) );
-			border-bottom: var( --wp-admin-border-width-focus ) solid ${ COLORS.theme.accent };
+			border-bottom: var( --wp-admin-border-width-focus ) solid theme.$components-color-accent;
 		}
 	}
+
 	&[aria-orientation='vertical'] {
+		flex-direction: column;
+
 		&::before {
 			/* Adjusting the border radius to match the scaling in the y axis. */
-			border-radius: ${ CONFIG.radiusSmall } /
+			border-radius: variables.$radius-small /
 				calc(
-					${ CONFIG.radiusSmall } / ( var( --selected-height, 0 ) / var( --antialiasing-factor ) )
+					variables.$radius-small / ( var( --selected-height, 0 ) / var( --antialiasing-factor ) )
 				);
 			top: 0;
 			left: 0;
 			width: 100%;
 			height: calc( var( --antialiasing-factor ) * 1px );
 			transform: translateY( calc( var( --selected-top, 0 ) * 1px ) )
 				scaleY( calc( var( --selected-height, 0 ) / var( --antialiasing-factor ) ) );
-			background-color: color-mix( in srgb, ${ COLORS.theme.accent }, transparent 96% );
+			background-color: color-mix( in srgb, theme.$components-color-accent, transparent 96% );
 		}
 		&[data-select-on-move='true']:has( :is( :focus-visible, [data-focus-visible] ) )::before {
 			box-sizing: border-box;
-			border: var( --wp-admin-border-width-focus ) solid ${ COLORS.theme.accent };
+			border: var( --wp-admin-border-width-focus ) solid theme.$components-color-accent;
 			/* Adjusting the border width to match the scaling in the y axis. */
 			border-block-width: calc(
 				var( --wp-admin-border-width-focus, 1px ) /
 					( var( --selected-height, 0 ) / var( --antialiasing-factor ) )
 			);
 		}
 	}
-`;
+}
 
-export const Tab = styled( Ariakit.Tab )`
+.tab {
+	/* stylelint-disable-next-line no-duplicate-selectors */
 	& {
```

Comment:
Was the `&` intended to be an Emotion specificity hack originally…? I don't think a single `&` like this has any effect (at least in Sass it won't).

Reply:
Probably, and you're right that right now, it's not making a difference. I'll remove it, and we can add it back if really needed (although I hope not).